lib/system_themes_manager.rb

```diff
@@ -26,6 +26,7 @@ def self.sync_theme!(theme_name)
         .first
         &.update!(user_selectable: true)
     end
+    remote_theme.update_column(:enabled, true)
```

Comment:
IMO stick to `update!(enabled: true)` here unless there is a reason not to do validations etc in this case, we already use `update!` above.